### ISSUE TITLE
There should/will be a feedback channel about relevancy and usefulness of information in the knowledge base (content). 

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#125]
_From @czdiana2321 on June 15, 2018 17:34_

_Copied from original issue: Microsoft/Access2Justice#155_

**Comment by KellyeDC-zz** [comment]
Is this a valid requirement? @breeglej @grawdon

**Comment by grawdon** [comment]
This is probably a Carlos question. I'm assuming they'll want something like this for their evaluation of the usefulness of the system and the states would want it to help see what content might need to be revised.

**Comment by breeglej** [comment]
Agree.  This needs to be something the evaluation team prescribes if necessary.
